examples/sensors/filter-with-expressions.yaml

```diff
@@ -2,9 +2,13 @@
 #
 #  {
 #    "a": "b",
-#    "c": 10,
-#    "d": {
-#      "e": false
+#    "a-longer-name": 10,
+#    "nested": {
+#      "path": {
+          "can-get": {
```

Comment:
Missing `#` for these lines, test case failing.

Reply:
Oh, sure. Thank!